packages/fetchai/protocols/contract_api/__init__.py

```diff
@@ -17,7 +17,12 @@
 #
 # ------------------------------------------------------------------------------
 
-"""This module contains the support resources for the contract_api protocol."""
+"""
+This module contains the support resources for the contract_api protocol.
+
+It was created with protocol buffer compiler version `libprotoc 3.11.4
+` and aea version `1.0.0rc1`.
```

Comment:
The reason why we kept the exact version was to be able to recreate the exact protocol. By committing this information here a developer can recreate the exact protocol folder when using the relevant versions.